be/src/exprs/vectorized/cast_expr.cpp

```diff
@@ -347,6 +348,33 @@ static ColumnPtr cast_from_string_to_hll_fn(ColumnPtr& column) {
 }
 CUSTOMIZE_FN_CAST(TYPE_VARCHAR, TYPE_HLL, cast_from_string_to_hll_fn);
 
+template <PrimitiveType FromType, PrimitiveType ToType, bool AllowThrowException>
+static ColumnPtr cast_from_string_to_bitmap_fn(ColumnPtr& column) {
+    ColumnViewer<TYPE_VARCHAR> viewer(column);
+    ColumnBuilder<TYPE_OBJECT> builder(viewer.size());
+    for (int row = 0; row < viewer.size(); ++row) {
+        if (viewer.is_null(row)) {
+            builder.append_null();
+            continue;
+        }
+
+        auto value = viewer.value(row);
+
+        BitmapValue bitmap;
+        if (bitmap.deserialize(value.data, value.size)) {
```

Comment:
why not use `roaring_bitmap_portable_deserialize_size` to fast check

Reply:
> why not use `roaring_bitmap_portable_deserialize_size` to fast check

ok, I will use it to remove allocate memory dynamically.